WireAPI/Package.resolved

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
hmm not sure this was needed, I guess it comes from running `swift package resolve`

Reply:
Try delete deleting it?

Reply:
I think @johnxnguyen removed it as it was not needed for building the project..